corehq/toggles/__init__.py

```diff
@@ -563,6 +563,30 @@ def toggles_enabled_for_user(username):
     }
 
 
+@quickcache(["email"], timeout=24 * 60 * 60, skip_arg=lambda _: settings.UNIT_TESTING)
```

Comment:
With a timeout like this, we probably want explicit validation, otherwise when you enable a toggle, you'd have to wait 24 hours for it to take effect.

Reply:
Just took a look at the code - didn't realize this was following an existing pattern (I hadn't seen `NAMESPACE_EMAIL_DOMAIN` before).  Those are cleared here:
https://github.com/dimagi/commcare-hq/blob/4c8bbe4093fc3501899480813ead8cb58c2222dd/corehq/apps/toggle_ui/views.py#L286-L294

Reply:
Good catch, thank you. Fixed in cfe40281c42 and updated the function name to match the namespace constant.